Automattic/wp-calypso
# Marketing: Update order of services

#### Changes proposed in this Pull Request

- Update ordering of services in the Marketing Hub (I think Looka would be great to put up top since it's relatively easy to complete and something that pertains to branding their sites).
- Fixed margins around logos within each card.
- Use actual Google Adwords logo instead of an illustration.

**Before:**
![Screenshot 2019-05-20 10 33 01](https://user-images.githubusercontent.com/4924246/58041444-f7d03a00-7aec-11e9-8465-5568370ae1c6.png)

**After:**
![Screenshot 2019-05-20 10 46 55](https://user-images.githubusercontent.com/4924246/58041447-facb2a80-7aec-11e9-947d-6e6f35585e40.png)


#### Testing instructions

1. Go to Marketing from the sidebar under Tools.
2. Check if above changes are reflected.

cc @huguesvincent  @belcherj @stephanethomas
